### PR TITLE
[Merged by Bors] - ET-4233 build services for arm

### PR DESCRIPTION
**Pull request by Robert-Steiner**
[ET-4233]

[ET-4233]: https://xainag.atlassian.net/browse/ET-4233?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/scripts/download_assets.sh [hunk 1]
```diff
@@ -1,46 +1,39 @@
...
-}
-
-# path to the directory where this file is
-SELF_DIR_PATH="$(dirname "$0")"
-
-# a parameter for the destination of the assets can be passed.
-# the default is the directory assets, we assume the script is in .github/scripts/
-DATA_DIR="${1:-$SELF_DIR_PATH/../../assets}"
-DATA_DIR=`realpath $DATA_DIR`
-
```
Comment (acrrd) [inline .github/scripts/download_assets.sh:15]: :wrench: 
This code was ensuring that the assets are always downloaded in the `assets` directory at the top level of the project. With this change, they will be downloaded in the `../../assets` directory depending from where I run the script.

File: .github/scripts/download_assets.sh [hunk 2]
```diff
@@ -1,46 +1,39 @@
...
+        curl "$BASE_URL/$1/$ARCHIVE_NAME" -o "$DATA_DIR/$TMP_ARCHIVE_NAME" -C -
+        mv "$DATA_DIR/$TMP_ARCHIVE_NAME" "$DATA_DIR/$ARCHIVE_NAME"
+
+        cd "$DATA_DIR"
+        tar -zxf "$ARCHIVE_NAME"
+
+        # check content
+        cd "$ARCHIVE_BASENAME"
+        shasum -c "$CHECKSUM_FILE"
+    fi
```
Comment (acrrd) [inline .github/scripts/download_assets.sh:26]: :wrench: 
I don't see any advantage or relevant change in this code w.r.t. to the previous one.

File: .github/workflows/ingestion-service.yml [hunk 3]
```diff
@@ -23,30 +30,53 @@ jobs:
...
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
 
       - name: Setup job
         uses: ./.github/actions/setup-job-docker
 
-      - name: Build artifacts
-        run: just build-ingestion-service ${{ github.event.inputs.features }}
+      - name: Build default artifacts
+        if: ${{ github.event.inputs.target == 'default' }}
+        run: just build-service ingestion "${{ github.event.inputs.features }}"
```
Comment (acrrd) [inline .github/workflows/ingestion-service.yml:43]: :question: Why are we interested in keeping the x86 image?

File: .github/scripts/download_assets.sh (hunk 2 above)
Comment (Robert-Steiner) [reply .github/scripts/download_assets.sh:26]: I don't know why that change was made and i don't have the time to look into this. maybe we can revert those changes and move them into a separate pr?

File: .github/workflows/ingestion-service.yml (hunk 3 above)
Comment (Robert-Steiner) [reply .github/workflows/ingestion-service.yml:43]: i will remove the x86 image

**Review by acrrd — APPROVED** [review]

**Comment by Robert-Steiner** [comment]
bors merge

**Pull request closed without merging**